docs/src/database_import.md

```diff
@@ -219,8 +219,6 @@ There are a few things you need to be aware of when using the `monolith` type:
   ["The `externalClusters` section"](bootstrap.md#the-externalclusters-section))
 - Traffic must be allowed between the Kubernetes cluster and the
   `externalCluster` during the operation
-- You need to specify `sslmode: disable` in the `connectionParameters` section
```

Comment:
we are using pgx but  still using the simple protocol, I think the same limitations still apply